### PR TITLE
Time selector - new IA

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/time-selector.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
```
Comment (palindromebart) [inline _component_design/time-selector.md]: thinking aloud, if you add the content from above, there'll be duplicative content in the BP section. so, you know, I trust you to not do that. 
lolz.

File: _component_design/time-selector.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A time selector is comprised of the following elements:
+
+- **Drop-down selector (required)**: All possible minute and hour combinations appear on a drop-down list that the user accesses by clicking the drop-down selector. The drop-down selector can be activated by keyboard input or when the user clicks anywhere on the time selector. See [drop-down selector]({{site.baseurl}}\components\drop-down-selector.html) for construction guidelines.
```
Comment (palindromebart) [inline _component_design/time-selector.md]: "all possible" opens the door for a massively crazy long list. maybe remove the all encompassing nature of the sentence and just state the the dropdown selector contains selectable times.

File: _component_design/time-selector.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Time zones
+
+Rackspace is a global company with users around the world.
+
+If you need to show a time zone, use Coordinated Universal Time (UTC), and indicate the time-zone offset from UTC. For example, use **10:30 (UTC -6)** (refers to CT).
```
Comment (palindromebart) [inline _component_design/time-selector.md]: should we add a link to text conventions date and time guidelines?

File: _component_design/time-selector.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
```
Comment (palindromebart) [inline _component_design/time-selector.md]: add a link to text conventions dat & time for best practices in time display

File: _component_design/time-selector.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
+---
+title: Time Selector
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time selector enables the user to select a time from a list of values. The time selector is often accompanied by a date selector that a user can use to select a date.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/time-selector.md]: For Time Input, wouldn't we have another page for that component? Or do we envision adding it to this doc?

File: _component_design/time-selector.md (hunk 1 above)
Comment (chri2547) [reply _component_design/time-selector.md]: All set on this one!

File: _component_design/time-selector.md (hunk 2 above)
Comment (chri2547) [reply _component_design/time-selector.md]: Gotcha! Done.

File: _component_design/time-selector.md (hunk 4 above)
Comment (chri2547) [reply _component_design/time-selector.md]: Yeppers, another good place for a xref link. Done, son!

File: _component_design/time-selector.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/time-selector.md]: We decided to leave this as is for now, and address this when we think about the combo box version in 2019.

File: _component_design/time-selector.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Time zones
+
+Rackspace is a global company with users around the world.
+
+If you need to show a time zone, use Coordinated Universal Time (UTC), and indicate the time-zone offset from UTC. For example, use **10:30 (UTC -6)** (refers to CT). See [Date and Time]({{site.baseurl}}/style/date-time.html) for more information on formatting time zones.
```
Comment (stephamon) [inline _component_design/time-selector.md]: I would expand the parenthetical here from (refers to CT) to (refers to U.S. Central Time)

Comment (chri2547) [reply _component_design/time-selector.md]: Love. It. Done.

File: _component_design/time-selector.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector when you want to enable a user to select a time from a list of values. The time selected includes an hour value and a minute value. The hour value can include 12-hour clock values, or 24-hour clock values. Minutes are grouped into 10-minute, 15-minute, or 30-minute intervals. For example, a user can select a time of **1:00 PM**, **1:10 PM**, **1:15 PM**, or **1:30 PM**, but cannot select any other minute values.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/time-selector.md]: Right-o, moved it and reworded BP content.

File: _component_design/time-selector.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector when you want to enable a user to select a time from a list of values. The time selected includes an hour value and a minute value.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
```
Comment (araiford) [inline _component_design/time-selector.md]: Our logic is a little muddled in the `When to Use` section.  The first sentence says that the selector should be used when you want the user to select from a list of values, but then the third sentence says that if you want the user to select a time from a limited list of predetermined times, to consider using radios.

If we don't want implementors to use the time selector for a set of limited, predetermined values, then we should be explicit in the first sentence and say something along the lines of  `Use the time selector to enable a user to choose a specific hour in the 24-hour range.`

File: _component_design/time-selector.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector to enable a user to choose a specific hour in the 24-hour range. The time selected includes an hour value and a minute value.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
```
Comment (nimbinatus) [inline _component_design/time-selector.md]:
```suggestion
If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}/components/radio-buttons.html).
```

File: _component_design/time-selector.md [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A time selector is comprised of the following elements:
+
+- **Drop-down selector (required)**: Selectable minute and hour combinations appear on a drop-down list that the user accesses by clicking the drop-down selector. The drop-down selector can be activated by keyboard input or when the user clicks anywhere on the time selector. See [drop-down selector]({{site.baseurl}}\components\drop-down-selector.html) for construction guidelines.
```
Comment (nimbinatus) [inline _component_design/time-selector.md]:
```suggestion
- **Drop-down selector (required)**: Selectable minute and hour combinations appear on a drop-down list that the user accesses by clicking the drop-down selector. The drop-down selector can be activated by keyboard input or when the user clicks anywhere on the time selector. See [drop-down selector]({{site.baseurl}}/components/drop-down-selector.html) for construction guidelines.
```

File: _component_design/time-selector.md (hunk 9 above)
Comment (chri2547) [reply _component_design/time-selector.md]: Weird, I coulda sworn I changed the name of radio-buttons to just radios. Looks like I did not do that. Accepted.

**Pull request merged**